dips/tests/test_helpers.py

```diff
@@ -51,3 +62,124 @@ def test_update_instance_from_dict(self):
+    def test_get_page_from_search_orm_defaults(self, patch, ):
+        # ORM QuerySet
+        page = helpers.get_page_from_search(self.orm_query_set, {})
+        self.assertEqual(page.paginator.count, 100)
+        self.assertEqual(page.paginator.num_pages, 10)
+        self.assertEqual(page.number, 1)
+        # The objects_list is the QuerySet SQL query
+        self.assertEqual(page.object_list.query.low_mark, 0)
+        self.assertEqual(page.object_list.query.high_mark, 10)
+
+    @patch('elasticsearch_dsl.Search.count', return_value=100)
+    def test_get_page_from_search_es_with_params(self, patch):
+        page = helpers.get_page_from_search(
+            self.es_search, {'page': '2', 'limit': '20'})
+        self.assertEqual(page.number, 2)
+        self.assertEqual(page.object_list.to_dict(), {'from': 20, 'size': 20})
+
+    @patch('django.db.models.query.QuerySet.count', return_value=100)
+    def test_get_page_from_search_orm_with_params(self, patch):
+        page = helpers.get_page_from_search(
+            self.orm_query_set, {'page': '2', 'limit': '20'})
+        self.assertEqual(page.number, 2)
+        self.assertEqual(page.object_list.query.low_mark, 20)
+        self.assertEqual(page.object_list.query.high_mark, 40)
+
+    @patch('elasticsearch_dsl.Search.count', return_value=100)
+    def test_get_page_from_search_es_wrong_page(self, patch):
+        # Not an integer
+        page = helpers.get_page_from_search(self.es_search, {'page': 'text'})
+        self.assertEqual(page.paginator.num_pages, 10)
+        self.assertEqual(page.number, 1)
+        self.assertEqual(page.object_list.to_dict(), {'from': 0, 'size': 10})
+        # Empty page
+        page = helpers.get_page_from_search(self.es_search, {'page': '11'})
+        self.assertEqual(page.paginator.num_pages, 10)
+        self.assertEqual(page.number, 10)
+        self.assertEqual(page.object_list.to_dict(), {'from': 90, 'size': 10})
+
+    @patch('django.db.models.query.QuerySet.count', return_value=100)
+    def test_get_page_from_search_orm_wrong_page(self, patch):
+        # Not an integer
+        page = helpers.get_page_from_search(
+            self.orm_query_set, {'page': 'text'})
+        self.assertEqual(page.paginator.num_pages, 10)
+        self.assertEqual(page.number, 1)
+        self.assertEqual(page.object_list.query.low_mark, 0)
+        self.assertEqual(page.object_list.query.high_mark, 10)
+        # Empty page
+        page = helpers.get_page_from_search(
+            self.orm_query_set, {'page': '11'})
+        self.assertEqual(page.paginator.num_pages, 10)
+        self.assertEqual(page.number, 10)
+        self.assertEqual(page.object_list.query.low_mark, 90)
+        self.assertEqual(page.object_list.query.high_mark, 100)
+
+    @patch('elasticsearch_dsl.Search.count', return_value=100)
+    def test_get_page_from_search_es_wrong_limit(self, patch):
+        wrong_limits = [
```

Comment:
Since these are used in multiple functions they could be moved to a class or module level var.